src/main/index.ts

```diff
@@ -113,16 +119,12 @@ async function initialize() {
     optimizer.watchWindowShortcuts(window)
   })
 
+  await Promise.all([setupBsAggregator(), app.whenReady()])
```

Comment:
What do you think?
```suggestion
  await Promise.allSettled([setupBsAggregator(), app.whenReady()])
```

Reply:
We can't open the app if any of these promises fail, so I can't use `allSettled`